### DISCUSSION TITLE
Change Theme per page

### DISCUSSION THREAD

**Discussion by AlexHedley** [microsoft/fluentui-blazor#3046] [Q&A]
Is it possible to set the _Theme_ on a specific page?

I have a _Settings Page_ with the dropdown to for a user to select an Office Colour but say when I go to a specific page in my App I'd like to change the theme without user input, what do I need to do?

Is it a matter or just adding this to the page?

```html
<FluentDesignTheme OfficeColor="OfficeColor.Word" />
```

**Comment by vnbaaij** [comment]
If you der it on that specific page, you set it for the whole site. So you'd need to find a way to set it back as well

**Reply by AlexHedley** [reply]
Should that override the current setting with just the above code or is anything else needed?
